guides/line-management/probation.md

```diff
@@ -0,0 +1,89 @@
+# Probation at Made Tech
+
+When someone joins Made Tech, they're "on probation" for a short time.
+This means their notice period is shorter - usually one week.
+
+At the end of probation, people either pass or fail. Most people pass.
+Rarely, you'll extend probation, or pass someone early.
+
+This guidance is written for line managers, but may be useful for people on probation too.
+
+- [What happens during probation?](#what-happens-during-probation)
+- [Explaining how to pass probation](#explaining-how-to-pass-probation)
+- [One-to-one meetings during probation](#one-to-one-meetings-during-probation)
+- [Finishing probation](#finishing-probation)
+
+## What happens during probation?
+
+In the first week, explain probation to your direct report. Talk about what it is, and how to pass it.
+
+Each week, meet them one-to-one.
+Get feedback from their team, and reflect on it together.
+Talk often about whether they're on-track to pass.
+
+At the end of probation, pass or fail it, and explain what happens next.
+
+## Explaining how to pass probation
+
+Talk about what good looks like for your direct report's role. [The handbook role pages](../../roles/README.md) may help with this.
+It's good to spend the first two weeks talking about this.
+
+Explain that feedback is useful evidence on how they're performing.
+
+Tell them to ask their team for feedback. You can talk to their team directly to help get more feedback, if needed.
+
+### Worries about failing probation
+
+It can be stressful to be on probation. Explain early that:
+- Most people pass
+- Getting regular feedback means the result won't be a surprise
+- You will guide and support them
+
+## One-to-one meetings during probation
+
+We don't have guidance for one-to-ones in the handbook yet. In brief, ask your direct report about their week:
```

Comment:
I think we do have guidance now, which could be linked to instead

Reply:
As someone about to be doing some line managing for the first time here, linking to that would be very useful.

Reply:
Great point, shows how old this PR is :grimacing:

Reply:
Addressed in https://github.com/madetech/handbook/pull/404/commits/075deb2c3790b7c20daeb9d04617dbe516dad78d (and then links hopefully fixed in https://github.com/madetech/handbook/pull/404/commits/18d56126d092f1b476c3886c915a0b9c4c6698a6 :grimacing:)